src/client/settings/templates/settings-cache.html

```diff
@@ -14,7 +14,7 @@
 
         <md-input-container>
             <md-checkbox ng-model="settings.config.deleteTmpOnClose">
-                Clear Tmp Folder after closing app?
```

Comment:
as i generally agree with you @hadim on the state of the project and it's hackability, i don't think it's good practice to change strings in a functional commit.